operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1118,6 +1118,12 @@ ruler:
   service:
     annotations: {}
     labels: {}
+  
+  serviceAccount:
+    create: true
+    name:
+    annotations: {}
+    labels: {}
```

Comment:
can you add a comment explaining how this relates to the default service account? i.e. the general service account can be disabled and this can still be enabled

Reply:
Sure.